Makefile

```diff
@@ -27,31 +29,31 @@ $(BINARY):
 
 .PHONY: docker
 docker:
-	docker build -f Dockerfile -t quay.io/pusher/oauth2_proxy:latest .
+	docker build -f Dockerfile -t $(HUB)/oauth2_proxy:latest .
```

Comment:
You'll need to find and replace `HUB` as part of the renaming

Reply:
🤦🏽‍♂️I thought I have done that. Sorry. Fixed.